Instagram_App/Instagram_App/ViewControllers/search/HistoriCollectionViewCell.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  HistoriCollectionViewCell.swift
+//  Instagram_App
+//
+//  Created by coder on 24.10.2022.
+//
+
+import UIKit
+
+/// ячейка с нашими историями 
+final class HistoriCollectionViewCell: UICollectionViewCell {
+    
+    // MARK: IBOutlet
+    @IBOutlet private weak var historyImageView: UIImageView!
+    @IBOutlet private weak var historyLabel: UILabel!
+    
+    // MARK: Public MEthods
+    func configHistory(_ model: CollectionCell, index: Int) {
```

Comment:
```suggestion
    func configHistory(_ model: CollectionCell) {
```
метод должен принимать один параметр _ model: CollectionCell